src/stores/preferences.ts

```diff
@@ -0,0 +1,19 @@
+import { Course } from "./schedule";
+
+export type Professor = {
+    preferences: Preference[];
+    coursesCanTeach: Course[];
+    displayName: string;
+    teachingStatus: string;
+    requiredEquipment: string;
+    hasPEng: boolean;
+    fallTermCourses: number;
+    springTermCourses: number;
+    summerTermCourses: number;
+  };
+  
+  export type Preference = {
+    course: Course;
+    preferenceNum: number;
+  };
+  
```

Comment:
Pretty bizarre indentation here, are you using Prettier? I'm going to keep nagging about this until the code looks pretty :stuck_out_tongue:

Reply:
LOL I am but I forgot on this file sad :(

Reply:
updated

Reply:
You should configure it to format the file when you save. Not only will this keep everything formatted all the time, but you can copy/paste around severely mal-formatted code and everything will be fixed for you. Saves a ton of time.

Reply:
yeah that's a good idea haha